### ISSUE TITLE
BUG: Parsing error

### ISSUE THREAD

**Issue by martinskalicky** [recursivetree/seat-transport#12]
![](https://i.imgur.com/kDhTC1A.png)

**Comment by recursivetree** [comment]
yeah that's a new formatting I haven't encountered before. From where did you get that?

**Comment by recursivetree** [comment]
Just to provide some context, CCP uses different copy paste formats everywhere. Jut look at the code responsible for parsing from evepraisal. https://github.com/evepraisal/go-evepraisal/tree/master/parsers

**Comment by recursivetree** [comment]
You can get it to work by removing the group name

**Comment by martinskalicky** [comment]
This is from the personal assets so if people wanna do contracts remotely they wont be able to do it. This will be frequent. Can we make it working somehow?

**Comment by recursivetree** [comment]
![Bildschirmfoto 2023-02-18 um 22 17 35](https://user-images.githubusercontent.com/60423027/219899894-71771b93-3139-49f2-bd7e-b066b146b7b0.png)
well, it looks different to me

**Comment by recursivetree** [comment]
it's the personal assets window

**Comment by martinskalicky** [comment]
![Screenshot 2023-02-19 at 13 07 10](https://user-images.githubusercontent.com/46750448/219946917-ed92042c-93ce-443d-84c0-32a9a4c3f55a.png)

**Comment by martinskalicky** [comment]
You are able to hide some of the stuff. I might test if caregory is gone if the parsing will work. However this should be default if u did not do any changes.

**Comment by recursivetree** [comment]
I need to rework the parser anyways. I planned to do it since a long time, but new, small feature requests and bugs keep coming in almost every day :) With the current way parsing is handled, it is almost impossible to add more features.

On another note, If the group is hidden, it  isn't possible to reliably detect containers when they're renamed

**Comment by recursivetree** [comment]
for now, just make sure the columns like in my screenshot are visible

**Comment by martinskalicky** [comment]
I've created a guide to workaround this in case anyone will go into trouble.

**Comment by recursivetree** [comment]
That doesn't mean it won't happen once the new parsers are done! reopening

**Comment by recursivetree** [comment]
I'll be able to fix this soon, now that the new parser infrastructure is ready

**Comment by martinskalicky** [comment]
"Isogen	719951
Megacyte	1
Mexallon	47760
Nocxium	214598
Pyerite	121676
Tritanium	6060433
Zydrine	4"  
 
**Throwing:**  Error
You need to enter at least one item!

using DEV branch

**Comment by recursivetree** [comment]
Is this still an issue? if so, please send a parser report from the tool found under https:your-seat.com/treelib/debug/parsers
